admin/services/data-access/pupil.data.service.js

```diff
@@ -136,17 +136,20 @@ pupilDataService.getStatusCodes = async () => {
 /** SQL METHODS */
 
 /**
- * Fetch all pupils for a school by dfeNumber sorted by pupil name
+ * Fetch all pupils for a school by dfeNumber sorted by pupil last name
  * @param {number} dfeNumber
+ * @param orderColumn
+ * @param orderDirection
  * @return {Promise<results>}
  */
-pupilDataService.sqlFindPupilsByDfeNumber = async function (dfeNumber) {
+pupilDataService.sqlFindPupilsByDfeNumber = async function (dfeNumber, orderDirection = 'ASC', orderColumn = 'lastName') {
```

Comment:
remember to ensure that there is no way that a rogue value could be passed from the UI -> Controller -> Service -> Data Service.  otherwise you could have code injected.  Best to sanitise for allowed values